lib/riax/riax_coverage_fsm.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Riax.Coverage.Fsm do
```

Comment:
some module level doc would be good here

Reply:
Done!